xbmc/cores/VideoPlayer/DVDInputStreams/dvdnav/dvd_types.h

```diff
@@ -188,6 +135,11 @@ typedef enum {
   DVD_PARENTAL_LEVEL_None = 15
 } DVDParentalLevel_t;
 
+/* Language ID (ISO-639 language code) */
+typedef uint16_t DVDLangID_t;
```

Comment:
i don‘t think there‘s a technical difference. we prefer `using DVDLangID_t = uint16_t` instead of typedeffing.
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#127-type-aliases

otherwise looks good at a glance

Reply:
Forget any changes to this header, it's external to kodi (belongs to libdvdnav). It's only here because we wrap the lib.
Goal is to keep them in sync with what is provided by the library itself until we are at a point of finally dropping them from kodi.

Reply:
arf, yeah. cool